src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -26,16 +28,27 @@ If you're not yet a New Relic customer and are just getting started learning abo
 
 ## How our pricing works [#how-pricing-works]
 
-Our usage-based pricing model is simple, which allows you to accurately predict your billing. It bills on these two factors:
+Our usage-based pricing model is simple, which allows you to accurately predict your billing. It bills on these two primary factors:
```

Comment:
@zuluecho9 I'm not sure "primary" adds much here. I'd just leave it the way it was.